Application/Published/Upgrade-AdminApp/Deploy-EdFiOdsAdminApp.ps1

```diff
@@ -0,0 +1,220 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+#Requires -Version 3.0
+
+<#
+.SYNOPSIS
+    Deploys Admin App appliacation to an existing Azure resource group
+.DESCRIPTION
+	Deploys Admin App appliacation to an existing Azure resource group
+.PARAMETER ResourceGroupName
+	Existing resource group name
+.PARAMETER ResourceGroupLocation
+    Existing resource group location
+.PARAMETER AdminAppName
+    A friendly name to help identify AdminApp within Azure.  Must be no more than 64 characters long
+.PARAMETER AppInsightLocation
+	Existing app insight location, mostly same as resouce group location
+.PARAMETER ProductionApiUrl
+	Existing ODS/ API Url that Admin App can connect
+.PARAMETER EncryptionKey
+	Base64-encoded 256 bit key appropriate for use with AES encryption
+.PARAMETER TemplateFileDirectory
+	Points the script to the directory that holds the Ed-Fi ODS install templates.  By default that directory is the same as the one that contains this script.
+.PARAMETER AdminAppVersion
+	Admin app version to be deployed
+.PARAMETER Edition
+	Edition (Test, Release) of the Azure Deploy scripts to deploy.  Defaults to Release.
+.EXAMPLE
+	.\Deploy-EdFiOds.ps1 -ResourceGroupName "Ed-Fi-Ods-Resourcegroup" -ResourceGroupLocation "South Central US" -AdminAppName "AdminApp-Latest" -AppInsightLocation "South Central US" -ProductionApiUrl "https://edfiodsapiwebsite-production-yuw8iui32.azurewebsites.net"
+    Deploys the provided version of the AdminApp to the South Central US Azure region with the default instance name
+#>
+Param(
+
+	[string] 
+	[Parameter(Mandatory=$true)] 
+	$ResourceGroupName,
+
+	[ValidateSet("East US", "South Central US")]
```

Comment:
Why is this restricted to only these two resource group locations?

Reply:
@stephenfuqua 
These are the existing supported resource group location.

Reply:
What a strange restriction. I will ask Jason if it can be removed.

Reply:
never mind, I'm just going to ignore it for now.